logic/src/main/java/org/itsallcode/whiterabbit/logic/service/project/ProjectFileProvider.java

```diff
@@ -0,0 +1,8 @@
+package org.itsallcode.whiterabbit.logic.service.project;
+
+import java.nio.file.Path;
+
+public interface ProjectFileProvider
```

Comment:
Is this really necessary? In my opinion it makes the code more complicated and harder to understand.

If your goal is to simplify tests (i.e. avoid implementing unnecessary mocks), I would prefer to use mockito and mock only the required methods of the `Config` interface.

Reply:
Thanks - done.
Class comment had been provided already.
Great!

Reply:
Thanks for your question.
I thought a while about this before and now again.
And you're right: My approach was triggered by testing being hard to set up.

Is this really necessary? - no.
Does it make the code more complicated and harder to understand ?- yes, I agree. Sigh.

On the other hand I found it not very beautiful to pass a complex "config" to several services each being interested only in few members, hence creating more dependencies than necessary and using a unnecessarily large interface. 

Another option could be to let Config simply implement (or extend) interface ProjektFileProvider. 
Would you prefer this option?

Reply:
Yet another option: the config values don't change at runtime. We could just pass them to the constructor and avoid the provider. That's even simpler ;)

Reply:
Excellent idea, thanks!
Done.